### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'DEG2RAD')

### ISSUE THREAD

**Issue by john681611** [reaviz/reagraph#25]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior
Fails to load with 
![image](https://user-images.githubusercontent.com/10814889/170695943-d2698615-4c60-4975-bc62-bf6f5aadb827.png)

## Expected behavior
It works with an example component

## Minimal reproduction of the problem with instructions

* Use Create-react-app to create basic app
* npm i reagraph --save
* Modify App.js to dump in example
```js
import logo from './logo.svg';
import './App.css';
import React from 'react';
import { GraphCanvas } from 'reagraph'

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
        <GraphCanvas
    nodes={[
      {
        id: "n-1",
        label: "1"
      },
      {
        id: "n-2",
        label: "2"
      },
      {
        id: "n-3",
        label: "3"
      },
      {
        id: "n-4",
        label: "4"
      }
    ]}
    edges={[
      {
        id: "1->2",
        source: "n-1",
        target: "n-2",
        label: "Edge 1-2"
      },
      {
        id: "1->3",
        source: "n-1",
        target: "n-3",
        label: "Edge 1-3"
      },
      {
        id: "1->4",
        source: "n-1",
        target: "n-4",
        label: "Edge 1-4"
      }
    ]}
  />
      </header>
    </div>
  );
}

export default App;

```
* npm start
* observe failure in console while nothing is rendered

## Environment

<pre><code>
Libs:
- react version: ^18.1.0
- realayers version: ^1.0.2

Browser:
- [x] Chrome (desktop) version 102.0.5005.63
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: v18.2.0
- Platform:  Windows

Others:

</code></pre>

**Comment by amcdnl** [comment]
I believe there was a typo in the docs, can u look at the docs again and see if that fixes it?

**Comment by Araniela** [comment]
@amcdnl This issue isn't solved yet, sadly. You can reproduce it by generating a brand new react-app, install the reagraph dependency, and importing the GraphCanvas component as per README. 

When we do npm start, the following error message is displayed on the console:

![image](https://user-images.githubusercontent.com/74309184/176149610-35929574-4ac1-495a-8ac1-0147c86a8db9.png)

The code changes are as simple as this:

`
import logo from './logo.svg';
import './App.css';
import { GraphCanvas } from 'reagraph';

function App() {
  return (
    <GraphCanvas
    nodes={[
      {
        id: '1',
        label: '1'
      },
      {
        id: '2',
        label: '2'
      }
    ]}
    edges={[
      {
        id: '1->2',
        source: 'n-1',
        target: 'n-2',
        label: 'Edge 1-2'
      }
    ]}
  />
  );
}

export default App;
`

Please advise.

Thanks,
Sara

**Comment by Araniela** [comment]
@amcdnl @john681611 I just downgraded a few dependencies, and it is working now. There is definitely a bug with latest Reagraph that needs to be addressed. Can we please re open this issue?

Working dependencies:
`    "react": "18.0.0",
    "react-dom": "18.0.0",
    "react-scripts": "4.0.3",
    "reagraph": "1.0.0"`
    
Issue reproduced with:
`    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scripts": "5.0.1",
    "reagraph": "^1.1.0",`

**Comment by amcdnl** [comment]
Could you make a Github repo showing this? I've not been able to reproduce it in my local examples.

**Comment by Araniela** [comment]
https://github.com/Araniela/reagraph-issue-25

Just run npm start and check the console error. Thank you!

**Comment by flc** [comment]
this combination of the dependencies also works for me:
```
"react": "^18.2.0",
"react-dom": "^18.2.0",
"react-scripts": "4.0.3",
"reagraph": "^1.2.0",
```
and the change to react-scripts 5.x introduces the issue.

**Comment by amcdnl** [comment]
Ah - thats helpful. That explains why I wasn't seeing it, I am running CRA 4.x

**Comment by Araniela** [comment]
Hello! Thanks so much for having a look into this. As long as we can test the latest version of reagraph with the latest versions of react, that's the most important thing as it guarantees we won't lose any fixes / new releases. 

Have you found the problem? Are you working on a fix? If so, can you please provide an estimate?

Cheers,
Sara

**Comment by amcdnl** [comment]
Should be resolved in 2.0.4
